jupiterone/questions/questions.yaml

```
@@ -0,0 +1,15 @@
+---
+sourceId: managed:template
+integrationDefinitionId: '${integration_definition_id}'
+questions:
+  - id: integration-question-sentry
+    title: What kinds of questions will this integration support?
+    description:
+      TODO Every integration should contribute questions! Please be carefule
+      to replace this before deploying the integration to JupiterOne.
+    queries:
+      - name: good
+        query: |
+          find * with _integrationDefinitionId = '${integration_definition_id}'
+    tags:
+      - template
```

Do we actually have any questions we want to start asking based on your initial ingestion? If so, maybe we can add those.

In any case, let's make sure we don't accidentally deploy this template question
```suggestion
questions:
```

---

Good call out.  I added in two questions from the initial definition Sam provided that our ingested data should be able to answer.